README.md

```diff
@@ -3,9 +3,9 @@ GoPro Metadata Format Parser
 
 TLDR:
 
-1. `ffmpeg -y -i GOPR0001.MP4 -codec copy -map 0:m:handler_name:"	GoPro MET" -f rawvideo GOPR0001.bin`
-2. `gopro2json -i GOPR0001.bin -o GOPR0001.json`
-3. There is no step 3
+1. `ffmpeg -y -i GOPR0001.MP4 -codec copy -map 0:3 -f rawvideo GOPR0001.bin`
```

Comment:
The data is not always in 0:3 (on firmware v2.00 it is not)
Instead, this always gets it, but in the .md file the tab character before GoPro is often displayed as a space.
0:m:handler_name:"	GoPro MET" 
In my repo I solved it by separating this line from the rest: https://github.com/JuanIrache/gopro-utils